build/yamls/antrea-windows-containerd.yml

```diff
@@ -0,0 +1,299 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-priv.ps1: |
+    $ErrorActionPreference = "Stop";    
+    mkdir -force c:/var/log/antrea
+
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/') 
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    
+  Run-AntreaAgent.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/* c:/k/antrea/bin/
+    & "$mountPath/k/antrea/bin/antrea-agent.exe" --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=4
```

Comment:
You have copied the files from `$mountPath` to `c:\k\antrea`, but used the file in $mountPath when running antrea-agent, I thought you wanted to run the file in c:/k ? Otherwise, you don't need to copy the files.

Reply:
we only need antctl.exe in C:\k\antrea, the reason is we need the same path in e2e test.

Reply:
I didn't get your point, firstly we want to run `C:\k\antrea\antctl.exe` inside Windows Pod or on Windows Node host?? 2) Why can not copy the file in initContainers?

Reply:
In function waitForNetworkpolicyRealized we have "antctlCmd := fmt.Sprintf("C:/k/antrea/bin/antctl.exe get networkpolicy -S %s -n %s -T %s", networkpolicy, data.testNamespace, npOption)". If we don't copy the antctl.exe, the traceflow command will return 'file not exist' error.

Reply:
Thanks for the comments, I will add another patch to support different antctl.exe path for docker and containerd.

Reply:
Since we already support running antrea-agent inside container not on the host, I don't think t is needed to run antctl on the host any more. This is unlike docker, in which case antrea-agent is actually running on the host, which requires antctl to run on the host to ensure the connection to antctl is working as expected.